internal/querycoord/task.go

```diff
@@ -2300,6 +2306,26 @@ func (lbt *loadBalanceTask) globalPostExecute(ctx context.Context) error {
 					}
 				}(replica)
 			}
+
+			for _, segment := range segments {
+				wg.Add(1)
+				go func(segment *querypb.SegmentInfo) {
+					defer wg.Done()
+
+					segment.NodeID = -1
+					segment.NodeIds = removeFromSlice(segment.NodeIds, lbt.SourceNodeIDs...)
+					if len(segment.NodeIds) > 0 {
+						segment.NodeID = segment.NodeIds[0]
```

Comment:
Why you set the first element in NodeIds as NodeID?

Reply:
The NodeID may refer to an offline node, so set it to an online node here

Reply:
@yah01 @sunby is this comment resolved?